src/widgets/copilot/components/ChatInput/chatInput.scss

```
@@ -2,18 +2,24 @@
 
 .gooeyChat-chat-input {
   width: 100%;
-  background: transparent;
+  max-width: var(--gooey-message-content-max-width, 760px) !important;
   position: sticky;
   bottom: 0;
-  background: rgba(255, 255, 255, 0.2);
-  backdrop-filter: blur(7px);
-  -webkit-backdrop-filter: blur(7px);
+  // Opaque surface on purpose: a translucent background + backdrop-filter
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)